lab.R

```diff
@@ -188,24 +247,46 @@ mob_df = read_csv(file.path('data', 'mobility.csv'))
 
 #' 1. *This is just one way we could get at the relationship between stay-at-home in June and the peak of the outbreak in July.  What are some other approaches we might take?*
 #' 
-#' 
+#' i would think we should be more granular. like looking at weekly averages instead of monthly. we could also look rate of change rather than total and see 2 week lag correlates well.
 #' 
 
 #' 2. *Construct a dataframe `parks_june` that reports the mean level of "parks" mobility for each county in June 2020.  Call the variable `parks`.  (Just so the automatic checks know where to look.)  Note that the Google data has a lot of gaps for the `parks` type.  Use the `na.rm = TRUE` argument in `mean()` to handle missing values, and then filter out missing values.  The final dataframe should have three columns: county name, FIPS code, and `parks`.  And it should have one row for each county in the mobility data for which we have an estimate for "parks".*
 #' 
 
+parks_june = mob_df %>%
+  filter(type=='parks',
+  date>='2020-06-01',
+  date<='2020-06-30') %>%
+  group_by(county, fips) %>% 
+  summarize(parks=mean(pct_diff, na.rm = TRUE)) %>% 
+  ungroup() %>% 
+  filter(!is.na(parks))
+
 #' 3. *Construct a dataframe `cases_july` that reports the total level of new cases per 1 million residents of each county in July 2020.  (Don't worry about negative values.  I'm just asking you to do `sum(cases_per_pop)`.)  This dataframe should have three columns and one row for each county in the Covid-19 data.*
 #' 
 
+cases_july = covid_df %>%
+  filter(date>='2020-07-01',
+         date<='2020-07-30') %>%
```

Comment:
Double check the number of days in July
```suggestion
         date<='2020-07-31') %>%
```